Orleans.Streams.RabbitMqStreamProvider/ClientBuilderExtensions.cs

```diff
@@ -1,26 +1,67 @@
 ﻿using System;
+using Orleans.Configuration;
 using Orleans.Streaming;
 using Orleans.Streams.BatchContainer;
 
 namespace Orleans.Hosting
 {
     public static class ClientBuilderExtensions
     {
+        /// <summary>
+        /// Configure client to use RMQ persistent streams, using the <see cref="DefaultBatchContainerSerializer"/>.
+        /// </summary>
+        [Obsolete("Use 'UseRabbitMqSteams'")]
+        public static IClientBuilder AddRabbitMqStream(this IClientBuilder builder, string name, Action<ClusterClientRabbitMqStreamConfigurator<DefaultBatchContainerSerializer>> configure = null)
+        {
+            return UseRabbitMqStreams(builder, name, configure);
+        }
+
         /// <summary>
         /// Configure client to use RMQ persistent streams.
-        /// This version enables to inject a custom BacthContainer serializer.
         /// </summary>
-        public static IClientBuilder AddRabbitMqStream<TSerializer>(this IClientBuilder builder, string name, Action<ClusterClientRabbitMqStreamConfigurator<TSerializer>> configure) where TSerializer : IBatchContainerSerializer, new()
+        [Obsolete("Use 'UseRabbitMqSteams'")]
+        public static IClientBuilder AddRabbitMqStream<TSerializer>(this IClientBuilder builder, string name, Action<ClusterClientRabbitMqStreamConfigurator<TSerializer>> configure = null)
+            where TSerializer : IBatchContainerSerializer, new()
         {
-            configure?.Invoke(new ClusterClientRabbitMqStreamConfigurator<TSerializer>(name, builder));
-            return builder;
+            return UseRabbitMqStreams(builder, name, configure);
+        }
+
+        /// <summary>
+        /// Configure client to use RMQ persistent streams, using the <see cref="DefaultBatchContainerSerializer"/>.
+        /// </summary>
+        public static IClientBuilder UseRabbitMqStreams(this IClientBuilder builder, string name, Action<RabbitMqOptions> options)
+        {
+            return UseRabbitMqStream<DefaultBatchContainerSerializer>(builder, name, options);
```

Comment:
Is this a typo? Sometimes it seems to be plural and sometimes singular.
`The name 'UseRabbitMqStream' does not exist in the current context`

Reply:
The original methods (marked `[Obsolete]`) are singular. The new ones are plural